<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deposito de Transferencias</string>
  </property>
  <widget class="QComboBox" name="cbsexo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-- Seleccione una opcion</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>hombre</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>mujer</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="datefechanacimiento">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>440</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(221, 0, 111);</string>
   </property>
   <property name="text">
    <string>Los campos con asterisco (*) son obligatorios</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbtipo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-- Seleccione una opcion</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DNI - documento nacional de identidad</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PSS - pasaporte</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VISA</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnregistrarDeposito">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>600</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
border-radius:8px</string>
   </property>
   <property name="text">
    <string>Registrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Primer nombre *</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Primero apellido *</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tipo de documento *</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtsegundonombre">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Numero de documento *</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Handwriting</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 255, 255);</string>
   </property>
   <property name="text">
    <string>Deposito Bancario Internacional</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Segundo nombre</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkterminos">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>620</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Acepta los terminos y condiciones ?  *</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>410</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sexo *</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>410</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lugar de nacimiento *</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>510</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Motivo del giro *</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbmotivogiro">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>540</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-- Seleccione una opcion</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Honorarios/servicios tecnicos</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Remesas/giros</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Donaciones</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cblugarnacimiento">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>440</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-- Seleccione una opcion</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>510</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Monto *</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>751</width>
     <height>4</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>4</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(177, 177, 177);
color: rgb(172, 172, 172);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtprimerapellido">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>camussoni</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtmonto">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>540</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtsegundoapellido">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/xy/banco/logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fecha de nacimiento *</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Segundo apellido</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtprimernombre">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>leonardo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtdocumento">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>38819232</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
